app/services/draw_starter.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+#
+# Service object to handle the starting of a draw. Checks to make sure the draw
+# is in the correct status, updates the status, and sends e-mail invitations to
+# students in the draw.
+class DrawStarter
+  # Class method to permit calling :start on the class without instantiating the
+  # service object directly
+  #
+  # @attr draw [Draw] the draw in question
```

Comment:
I think we should just enforce instantiation or make this a module instead of a class